### PR TITLE
add AKS rotateClusterCertificates API

### PR DESCRIPTION
**Pull request by andyzhangx**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

/hold

### CODE REVIEW

**Comment by andyzhangx** [comment]
hold on this PR until original AKS rotate-certs code merged into AKS code branch.

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json [hunk 1]
```diff
@@ -921,6 +921,50 @@
...
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
```
Comment (KrisBash) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json]: Is there no response body to the action?

**Review by KrisBash — APPROVED** [review]
Signing off with comment. If there are response bodies for the action, please model those in swagger.

**Comment by andyzhangx** [comment]
cc @yangl900 @weinong for review

**Comment by jhendrixMSFT** [comment]
@andyzhangx hello any update?

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json (hunk 1 above)
Comment (andyzhangx) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json]: I have removed 200 response, there is only 202

**Comment by andyzhangx** [comment]
> @andyzhangx hello any update?

the `rotateClusterCertificates` API has been merged in AKS, and I have removed 200 response in the PR. Why is this PR failing? @jhendrixMSFT

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json (hunk 1 above)
Comment (andyzhangx) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json]: now I use 200 and 204 instead.
```
A 'POST' operation 'ManagedClusters_RotateClusterCertificates' with x-ms-long-running-operation extension must have a valid terminal success status code 200 or 201 or 204
```

**Comment by andyzhangx** [comment]
@jhendrixMSFT I have updated this PR, pls take a review, thanks.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**